core/config/docs/core.toml

```diff
@@ -509,6 +509,28 @@ DeltaReconcile = '1m' # Default
 # but the host and port must be fully specified and cannot be empty. You can specify `0.0.0.0` (IPv4) or `::` (IPv6) to listen on all interfaces, but that is not recommended.
 ListenAddresses = ['1.2.3.4:9999', '[a52d:0:a88:1274::abcd]:1337'] # Example
 
+[Capabilities.WorkflowConnectorConfig]
+# ChainIDForNodeKey is the ChainID of the network
+ChainIDForNodeKey = '1' # Example
```

Comment:
As discussed before, you'll also need to provide the desired address of a key you want to use. You can handle it in a separate PR but it will save you some trouble if you don't have to re-generate all test outputs again.

Reply:
I've generated `0xfd29Dd9C980D715a64dace97F7A2AB98bcaE0fed` so I'll use that as the default.